inc/namespace.php

```diff
@@ -56,17 +56,34 @@ function get_merged_config() : array {
 
 	// Get custom config overrides.
 	$composer_file = ROOT_DIR . '/composer.json';
-	$composer_json = get_json_file_contents_as_array( $composer_file );
 
+	$composer_json = get_json_file_contents_as_array( $composer_file );
+	$environment = get_environment_type();
+	validate_config_settings( $composer_json, $environment );
 	$config = merge_config_settings( $default_config, $composer_json['extra']['altis'] ?? [] );
 
 	// Look for environment specific settings in the config and merge it in.
-	$environment = get_environment_type();
 	$config = merge_config_settings( $config, $config['environments'][ $environment ] ?? [] );
 
 	return $config;
 }
 
+function validate_config_settings( $composer_json, $environment ) {
```

Comment:
Try and get in the habit of using type hints with your functions, helps to catch errors and makes development clearer and easier.

```suggestion
function validate_config_settings( array $composer_json, string $environment ) {
```